generators/embeddable-graphic/templates/src/js/scripts.js

```diff
@@ -0,0 +1,25 @@
+import $ from 'jquery';
+import pym from 'pym.js';
+
+
+const pymChild = new pym.Child();
+
+$(() => {
```

Comment:
Also wondering if this file wouldn't benefit from some boilerplate d3 structure. Margins, dimensions and appending an SVG, for example.